brave/brave-core
# Fix missing Chromium l10n strings.

Fixes brave/brave-browser#12904

Updated pulll10n script to check out original Chromium GRD*s and XTBs before pulling from Transifex.
Re-ran pull_l10n.

## Submitter Checklist:


## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on.

## Test Plan:

1. Start Brave
2. Show bookmarks bar (Ctrl + Shift + B)
3. Right click on the bookmarks bar and add a new folder.
4. Bookmark several pages and save bookmarks in the folder created in the above step.
5. In settings switch the browser UI language to German.
6. Right click on the bookmarks folder created in step 3.
7. Verify that all items in the context menu are localized. 

Additionally, treat this as a l10n pull and per prior @kjozwiak's instructions (#10735 (comment)):

@brave/legacy_qa should run through a few locals just to make sure that nothing obvious has regressed. Shouldn't spend too much time on this though - can be combined with testing of https://github.com/brave/brave-browser/issues/13040
